test/integration/scheduler/scheduler_test.go

```diff
@@ -74,166 +73,6 @@ var _ = ginkgo.Describe("Scheduler", func() {
 		spotUntaintedFlavor = testing.MakeResourceFlavor("spot-untainted").Label(instanceKey, "spot-untainted").Obj()
 	})
 
-	ginkgo.When("Scheduling v1.Job", func() {
-		var (
-			prodClusterQ *kueue.ClusterQueue
-			devClusterQ  *kueue.ClusterQueue
-			prodQueue    *kueue.Queue
-			devQueue     *kueue.Queue
-		)
-
-		ginkgo.BeforeEach(func() {
-			gomega.Expect(k8sClient.Create(ctx, onDemandFlavor)).Should(gomega.Succeed())
-			gomega.Expect(k8sClient.Create(ctx, spotTaintedFlavor)).Should(gomega.Succeed())
-			gomega.Expect(k8sClient.Create(ctx, spotUntaintedFlavor)).Should(gomega.Succeed())
-
-			prodClusterQ = testing.MakeClusterQueue("prod-cq").
-				Cohort("prod").
-				Resource(testing.MakeResource(corev1.ResourceCPU).
-					Flavor(testing.MakeFlavor(spotTaintedFlavor.Name, "5").Max("5").Obj()).
-					Flavor(testing.MakeFlavor(onDemandFlavor.Name, "5").Obj()).
-					Obj()).
-				Obj()
-			gomega.Expect(k8sClient.Create(ctx, prodClusterQ)).Should(gomega.Succeed())
-
-			devClusterQ = testing.MakeClusterQueue("dev-clusterqueue").
-				Resource(testing.MakeResource(corev1.ResourceCPU).
-					Flavor(testing.MakeFlavor(spotUntaintedFlavor.Name, "5").Obj()).
-					Flavor(testing.MakeFlavor(onDemandFlavor.Name, "5").Obj()).
-					Obj()).
-				Obj()
-			gomega.Expect(k8sClient.Create(ctx, devClusterQ)).Should(gomega.Succeed())
-
-			prodQueue = testing.MakeQueue("prod-queue", ns.Name).ClusterQueue(prodClusterQ.Name).Obj()
-			gomega.Expect(k8sClient.Create(ctx, prodQueue)).Should(gomega.Succeed())
-
-			devQueue = testing.MakeQueue("dev-queue", ns.Name).ClusterQueue(devClusterQ.Name).Obj()
-			gomega.Expect(k8sClient.Create(ctx, devQueue)).Should(gomega.Succeed())
-		})
-
-		ginkgo.AfterEach(func() {
-			gomega.Expect(framework.DeleteNamespace(ctx, k8sClient, ns)).To(gomega.Succeed())
-			framework.ExpectClusterQueueToBeDeleted(ctx, k8sClient, prodClusterQ, true)
-			framework.ExpectClusterQueueToBeDeleted(ctx, k8sClient, devClusterQ, true)
-			framework.ExpectResourceFlavorToBeDeleted(ctx, k8sClient, onDemandFlavor, true)
-			gomega.Expect(framework.DeleteResourceFlavor(ctx, k8sClient, spotTaintedFlavor)).To(gomega.Succeed())
-			gomega.Expect(framework.DeleteResourceFlavor(ctx, k8sClient, spotUntaintedFlavor)).To(gomega.Succeed())
-		})
-
-		ginkgo.It("Should schedule jobs as they fit in their ClusterQueue", func() {
```

Comment:
We should have a version of this test with just workloads.

Reply:
Added